### PR TITLE
Update CFV to prepare for Dec 2 launch

### PR DESCRIPTION
**Pull request by worldwise001**

### CODE REVIEW

File: _pages/volunteer.md [hunk 1]
```diff
@@ -1,278 +1,231 @@
...
+At the same time, we realize that not every role will be suitable for every volunteer. For each role, we will mention possible gotchas in order to provide an inclusive experience for all our volunteers. This year, we are adding the following classifications to our roles:
 
-**Brief description:**
+- Roles that require lots of interactions with people
+- Roles that may have high noise/sensory effects
+- Roles that require mobility
+- Roles that may require standing
+- Roles that require strength
+- URM-only shifts: women/nonbinary, LGBTQ+, PoC
+-
```
Comment (bsideslauren) [inline _pages/volunteer.md]: Extra bullet

Comment (worldwise001) [reply _pages/volunteer.md]: fixed. thanks!

**Comment by bsideslauren** [comment]
Might make sense to update the landing page as well (index.html) announcing the CFV.

**Comment by bsideslauren** [comment]
We'll also need to update site nav with links to volunteer site. We can either do that in this PR or do that after the page/CFV is live.

**Comment by bsideslauren** [comment]
We can do the two items I mentioned in the above comments after we commit this PR as they mainly help with discoverability of the volunteers page.

**Comment by worldwise001** [comment]
Oh I can add those as well. Should be no big deal.

**Pull request merged**